Karma/Models/PostsData.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Karma.Models
+{
+    public class PostsData
+    {
+        public static double AvaragePost;
```

Comment:
We'll need to fix that 'a' in 'Average' next time😆

Reply:
Will do that now.